README.md

```diff
@@ -47,7 +47,7 @@ datasources:
     # proxy or direct (Server or Browser in the UI).
     access: proxy
     # <string> Sets URL for sending queries to VictoriaLogs server.
-    # see https://docs.victoriametrics.com/victorialogs/querying/
+    # see {{% ref "./querying/README.md" %}}
```

Comment:
let's leave documentation absolute link, in that case people can navigate from this repo